quesma/queryparser/pancake_model.go

```diff
@@ -47,6 +61,14 @@ type pancakeModelBucketAggregation struct {
 	filterOurEmptyKeyBucket bool
 }
 
+type pancakeModelPipelineAggregation struct {
+	name         string // as originally appeared in Query DSL
+	internalName string // full name with path, e.g. metric__byCountry__byCity__population or aggr__byCountry
```

Comment:
Remove `internalName` since we don't generate SQL column.

Reply:
Can do that, but told you one counterargument. Waiting for your verdict 🙂